icons/theguardian.svg

```diff
@@ -0,0 +1 @@
78c-.231-.11-.5-.142-.76-.14Zm2.787.002c-.032.008-.065.008-.097.024l-1.066.196c-.033.008-.065 0-.073.024-.041.081.08.09.13.106a.992.992 0 0 1 .146.04c.017 0 .057.009.065.017.008.008 0 .049 0 .057v2.262c0 .016.008.065 0 .081-.016.033-.081.057-.114.073-.048.025-.154.057-.17.106 0 .008-.008.098 0 .106h.024c.032.016.106 0 .146 0h1.351c.016 0 .065.008.073 0a.27.27 0 0 0 0-.114c-.008-.016-.032-.025-.049-.033-.04-.024-.09-.048-.13-.073-.024-.008-.081-.024-.097-.049-.009-.016 0-.04 0-.057v-1.57c0-.033-.009-.09 0-.122.008-.033.016-.025.056-.04a.915.915 0 0 1 .253-.082c.187-.04.407-.049.602-.008v-.83c0-.016.008-.057 0-.073-.025-.041-.163-.025-.212-.025a.65.65 0 0 0-.17.04c-.31.123-.448.473-.521.774-.008.008-.017.114-.05.081v-.016c-.007-.024 0-.065 0-.09v-.716c0-.008.01-.065 0-.073-.007-.016-.08-.016-.097-.016zm5.76 0c-.032.008-.064.016-.105.008-.203.033-.399.073-.602.106-.106.016-.203.04-.31.057-.073.016-.146.016-.21.04-.009.009-.025 0-.033.009a.2.2 0 0 0 0 .097c.016.025.081.033.097.041.057.024.106.04.163.065.033.016.073.016.081.049.009.032 0 .09 0 .122v1.814c0 .122.017.26 0 .383 0 .032.009.024-.032.049-.049.024-.098.056-.147.08-.024.01-.08.025-.097.05-.016.016-.008.114.008.122.024.016.114 0 .146 0h1.237c.057 0 .122.008.17 0h.025c.008-.008 0-.106 0-.114-.008-.025-.057-.04-.073-.049-.049-.024-.09-.049-.138-.073-.033-.017-.065-.025-.073-.057-.009-.033 0-.082 0-.114v-2.547c0-.024.008-.081 0-.105-.017-.041-.066-.025-.106-.033zm-12.171.003a.272.272 0 0 0-.065.005c-.139.016-.285.049-.431.073l-.57.098c-.057.008-.122.008-.17.032-.009.009-.017 0-.025.009-.008.008 0 .073 0 .09.065.032.13.056.211.089.025.016.122.04.139.065.008.016 0 .065 0 .081v1.709c0 .162.016.333.09.488.097.22.292.35.52.399.31.065.643-.009.911-.171a1.28 1.28 0 0 0 .212-.155c.024-.016.057-.049.073-.024.016.024 0 .09 0 .114v.138c0 .016-.008.081 0 .098.008.008.057.008.073.008a.197.197 0 0 0 .098 0c.154-.017.317-.041.48-.057.187-.025.366-.041.545-.065.016-.009.098 0 .114-.017.008 0 .008-.057 0-.073-.008-.016-.016-.016-.04-.024-.082-.033-.172-.073-.253-.106-.04-.016-.033-.008-.04-.033-.009-.032 0-.073 0-.097V13.11c0-.089.016-.203 0-.284-.009-.041.007-.025-.033-.033a.31.31 0 0 0-.122 0c-.342.049-.684.122-1.017.18-.041.007-.114 0-.155.024-.008.008-.008 0-.016.008s-.008.081 0 .09c.008.016.065.024.081.032.049.016.09.04.138.057.025.008.106.032.13.057.009.016 0 .065 0 .081V15.315c0 .025.009.057 0 .082-.032.097-.203.163-.292.179-.114.024-.252.024-.35-.04-.195-.123-.138-.497-.138-.7v-1.701c0-.098.016-.212 0-.31-.009-.032.008-.024-.033-.032a.202.202 0 0 0-.065-.013zm9.126.136a.486.486 0 0 1 .288.064c.056.033.065.041.065.09v2.376a.473.473 0 0 1-.692-.106c-.163-.26-.17-.618-.179-.911-.008-.342 0-.692.098-1.017.049-.18.138-.4.325-.472a.361.361 0 0 1 .095-.024zm-5.302 1.325v1.205c-.106.122-.325.138-.456.048-.122-.08-.154-.244-.17-.374a1.262 1.262 0 0 1 .04-.578c.09-.268.342-.284.586-.3zm10.3 0v1.205c-.105.122-.325.138-.455.048-.122-.08-.155-.244-.171-.374a1.262 1.262 0 0 1 .04-.578c.09-.268.342-.284.586-.3z"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>The Guardian</title><path d="m9.272 8.129-1.16.207V8.4l.32.127v3.577l-.28.16v.103h1.52v-.103l-.264-.145V9.961a.64.64 0 0 1 .4-.129c.24 0 .368.112.368.408v1.871l-.272.145v.111h1.512v-.111l-.271-.145v-1.863c0-.536-.225-.816-.729-.816a1.54 1.54 0 0 0-.992.361l-.031.031V8.13zm-4.512.566-.047 1.121H4.8l.744-1.007h.318v3.23l-.416.217v.111H7.36v-.119l-.416-.217V8.81h.32L8 9.816h.111l-.047-1.12zm8.033.785c-.888 0-1.434.569-1.434 1.489 0 .936.505 1.478 1.48 1.478.553 0 .952-.303 1.104-.52v-.11c-.2.103-.414.24-.718.24-.632 0-.896-.457-.928-1.05h1.752v-.023c0-1.088-.432-1.504-1.256-1.504m-.033.12c.256 0 .385.488.385 1.224l-.866.047c.002-.715.218-1.271.48-1.271m2.577 1.896-1.217.217v.08l.328.111v.928c-.088-.048-.271-.088-.511-.088-.84 0-1.4.52-1.4 1.6 0 1.024.48 1.52 1.128 1.52a1.07 1.07 0 0 0 .8-.329h.032v.32l.096.016 1.201-.158v-.106l-.338-.119v-3.976zm1.367 0a.513.513 0 0 0-.527.512.51.51 0 0 0 .527.511c.288 0 .537-.215.537-.511 0-.288-.249-.512-.537-.512m-14.52.184C1.04 11.68 0 12.233 0 13.793c0 1.456.84 2.07 2.145 2.07a4 4 0 0 0 1.359-.23v-1.465l.367-.2v-.136H2.072v.12l.375.216v1.48a.8.8 0 0 1-.336.055c-.568 0-.806-.624-.814-2.008-.008-1.16.302-1.863.91-1.863.208 0 .314.016.434.072l.664 1.047h.111l-.023-1.088c-.248-.104-.721-.183-1.21-.183m20.714 1.033c-.36.016-.714.157-.954.375h-.039v-.367h-.12l-1.2.214v.122l.336.111v2.36l-.287.16v.113h1.566v-.114l-.271-.152v-2.264a.65.65 0 0 1 .408-.127c.248 0 .385.12.385.424v1.951l-.281.16v.114H24v-.113l-.28-.16v-2.008c0-.56-.231-.8-.751-.8zm-5.772.04-1.236.224v.096l.336.127v2.32l-.282.16v.114h1.56v-.113l-.27-.153v-2.76zM5 12.76l-1.2.217v.087l.329.143v1.8c0 .52.264.856.84.856a1.34 1.34 0 0 0 .902-.375h.033v.36l.12.007 1.193-.152v-.08l-.32-.135v-2.72l-.13-.008-1.199.217v.087l.328.143v2.129a.49.49 0 0 1-.392.183c-.232 0-.367-.087-.367-.375v-2.369zm3.545 0a3.5 3.5 0 0 0-1.098.191v.809h.088l.656-.88.13-.017c.367 0 .495.201.495.793v.393l-.007-.01-.649.129c-.624.12-.928.393-.928.897s.32.798.8.798c.375 0 .617-.127.777-.351h.03c.089.216.273.343.673.343.248 0 .472-.063.584-.127v-.08l-.287-.08v-1.92c0-.664-.464-.888-1.264-.888m2.879 0-.008.008-1.2.224v.08l.329.096v2.352l-.281.16v.113h1.56v-.113l-.271-.153v-1.734c.176-.08.382-.129.654-.129.088 0 .192 0 .248.016v-.897c-.032-.016-.094-.015-.15-.015-.344 0-.632.24-.76.88h-.026v-.88zm7.488 0c-.464 0-.816.103-1.096.191v.809h.088l.657-.88.127-.017c.36 0 .496.201.496.793v.393h-.008l-.64.127c-.633.12-.936.384-.936.888s.32.8.8.8c.376 0 .616-.128.776-.352h.023c.088.216.272.343.672.343.256 0 .482-.063.594-.127v-.08l-.29-.08v-1.92c0-.664-.463-.888-1.263-.888m-4.8.144c.2-.008.255.056.335.104v2.377a.42.42 0 0 1-.279.103c-.432.016-.584-.425-.584-1.209 0-.872.207-1.359.527-1.375m-5.295 1.287v1.186c-.056.064-.128.103-.248.103-.216 0-.377-.129-.377-.609 0-.44.104-.64.424-.664zm10.368 0v1.186h-.008c-.048.064-.12.103-.24.103-.225 0-.383-.129-.383-.609 0-.44.111-.64.431-.664z"/></svg>
```

This looks good on my end and is a lot smaller! Please take a look on your SVG editor and make sure it looks right/good as well.